binder/src/main/java/io/grpc/binder/BinderServerBuilder.java

```diff
@@ -53,17 +53,35 @@
     extends ForwardingServerBuilder<BinderServerBuilder> {
 
   /**
-   * Creates a server builder that will bind with the given name.
+   * Creates a server builder that will listen for bindings to the specified address.
+   *
+   * <p>The listening {@link IBinder} associated with new {@link Server}s will be stored
+   * in {@code binderReceiver} upon {@link #build()}. Callers should return it from {@link
+   * Service#onBind(Intent)} when the binding intent matches {@code listenAddress}.
+   *
+   * @param listenAddress an Android Service and binding Intent associated with this server.
+   * @param receiver an "out param" for the new {@link Server}'s listening {@link IBinder}
+   * @return a new builder
+   */
+  public static BinderServerBuilder forAddress(AndroidComponentAddress listenAddress,
+      IBinderReceiver receiver) {
+    return new BinderServerBuilder(listenAddress, receiver);
+  }
+
+  /**
+   * Creates a server builder that will listen for bindings to the specified Service with the
```

Comment:
Let's just remove this method since it doesn't serve much purpose anymore.

Reply:
Done.